### DISCUSSION TITLE
sync a specific folder or type of login

### DISCUSSION THREAD

**Discussion by johnnycash69** [blacs30/bitwarden-alfred-workflow#119] [Q&A]
Is it possible to sync just "logins" and nothing (identity, notes etc.) else?

**Comment by blacs30** [comment]
No it is not.
What's the idea/reason behind it? Attachments are never synced, only downloaded on requests. The payload shouldn't be too big.
The native Bitwarden CLI, which is currently used in the background, also syncs everything.

**Reply by johnnycash69** [reply]
I'm having a lot of software licenses (< 300) in my secure notes that I don't need in the workflow. So atm the result of my search is very confusing and distracting.
This could be solved if I could sync specific folders or types of items.

**Comment by blacs30** [comment]
Okay that makes sense. 
Actually I think it might be possible. 
Since I create a local encrypted cache https://github.com/blacs30/bitwarden-alfred-workflow/blob/master/src/cache.go#L18 it would be possible to specify the item type to sync (i.e. all not in the list will be dropped). 
Then only those items which are left will be displayed in the workflow search.

**Reply by johnnycash69** [reply]
Awesome. So the workflow needs to be updated or could I specify the item type somewhere in the workflow settings?

**Reply by blacs30** [reply]
Yes the workflow needs an update. It's a new feature to be implemented. So it will take some time until it will be available.

**Reply by johnnycash69** [reply]
Nice and thank you for your awesome work. Love the workflow 🥇

**Reply by blacs30** [reply]
@johnnycash69  with the [latest release](https://github.com/blacs30/bitwarden-alfred-workflow/releases/tag/2.4.2)  this is implemented (hopefully in the way that is useful to you and the community)

**Reply by johnnycash69** [reply]
> @johnnycash69 with the [latest release](https://github.com/blacs30/bitwarden-alfred-workflow/releases/tag/2.4.2) this is implemented (hopefully in the way that is useful to you and the community)

Many thanks. Works like a charm :)
